addons/packages/pinniped/0.12.1/fixtures/expected/mc-ldap-custom-cluster-issuer.yaml

```diff
@@ -0,0 +1,2647 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: dex-selfsigned-ca-issuer
+  namespace: tanzu-system-auth
+spec:
+  selfSigned: {}
```

Comment:
Unfortunately the `dex-selfsigned-ca-issuer` and `pinniped-selfsigned-ca-issuer` still get created even though we are using a `custom_cluster_issuer`.

See (https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/pinniped/0.12.1/bundle/config/overlay/dex-certs-selfsigned.yaml#L59-L64) and (https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/pinniped/0.12.1/bundle/config/overlay/pinniped-cert.yaml#L46-L51).